test/packages/good/elasticsearch/transform/good_transform_abc_1/default.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Could you please add also transforms referenced by Kevin in the other conversation?

Reply:
Done